Trigger/src/EvalWeightedEventCounts_module.cc

```diff
@@ -0,0 +1,180 @@
+  Hist._hOccInfo[index_last][0] =
+      occInfoDir.make<TH1F>(Form("hInstLum_%i", index_last),
+                            "distrbution of instantaneous lum; p/pulse", 1000, _minPOT, _maxPOT);
+  Hist._hOccInfo[index_last][1] =
+      occInfoDir.make<TH1F>(Form("hInstLum2B_%i", index_last),
+                            "distrbution of instantaneous lum re-weighted in 2 batch-mode; p/pulse",
+                            1000, _minPOT, _maxPOT);
+}
+
+//--------------------------------------------------------------------------------//
+void EvalWeightedEventCounts::beginJob() { bookHistograms(); }
+
+//--------------------------------------------------------------------------------//
+void EvalWeightedEventCounts::endJob() {
+
+  //    evalTriggerRate();  Tag names must be specified in the fcl
+  printf("[%s::enJob] totWg    = %10.3e\n", _tagName.c_str(), _wgSum[0]);
+  printf("[%s::enJob] totEvtB1 = %10.3f\n", _tagName.c_str(), _wgSum[1]);
+  printf("[%s::enJob] totEvtB2 = %10.3f\n", _tagName.c_str(), _wgSum[2]);
+}
+
+//--------------------------------------------------------------------------------
+
+//================================================================
+void EvalWeightedEventCounts::beginRun(const art::Run& run) {
+  // get bfield
+  // GeomHandle<BFieldManager> bfmgr;
+  // GeomHandle<DetectorSystem> det;
+  // CLHEP::Hep3Vector vpoint_mu2e = det->toMu2e(CLHEP::Hep3Vector(0.0,0.0,0.0));
+  // _bz0 = bfmgr->getBField(vpoint_mu2e).z();
+
+  // mu2e::GeomHandle<mu2e::Tracker> th;
+  // _tracker  = th.get();
+}
+
+//--------------------------------------------------------------------------------
+void EvalWeightedEventCounts::analyze(const art::Event& event) {
+
+  // get the number of POT
+  double lumi = -1.;
+  auto const evtWeightH = event.getValidHandle<ProtonBunchIntensity>(_evtWeightTag);
+  lumi = (double)evtWeightH->intensity();
+
+  // 1 batch mode
+  _wgSum[0] += lumi;
+
+  const static double mean_b1 = 1.6e7; // Mean Proton Pulse Intensity in 1-Batch Mode
+  const static double mean_b2 = 3.9e7; // Mean  Proton Pulse Intensity in 2-Batch Mode
+  const static double sigma = 0.7147;  // Standard deviation of the x*log-normal(x) of the Proton
+                                       // Pulse Intensity distribution
+  const static double mub1 = log(mean_b1) - 0.5 * sigma * sigma;
+  const static double mub2 = log(mean_b2) - 0.5 * sigma * sigma;
+  const static double xlognorm_norm_b1 =
+      6.293492e-8; // evaluated in ROOT for cut_off = 1.2e8,
+                   // above mub(1/2) and sigma for x*log-normal(x)
+  const static double xlognorm_norm_b2 = 2.887949e-8; // evaluated in ROOT
+  const static double cut_off_norm_b1 =
+      ROOT::Math::lognormal_cdf(1.2e8, mub1, sigma); // Due to max cutoff in generation
+  const static double cut_off_norm_b2 =
+      ROOT::Math::lognormal_cdf(1.2e8, mub2, sigma); // Due to max cutoff in generation
+  // if(mode == 1) {
```

Comment:
You no longer have a variable named "mode"; so remove the commented out if block.

Reply:
Removed the block.